beacon-chain/core/state/state_transition.go

```diff
@@ -1,6 +1,9 @@
 package state
```

Comment:
This file basically merges the computation of a new active state with a new crystallized state into a single function